### DISCUSSION TITLE
Organization Account Flagged 2 Weeks +

### DISCUSSION THREAD

**Discussion by khemraj-h** [community/community#53608] [Enterprise]
### Select Topic Area

Product Feedback

### Body

It beens over 2 weeks since my organization account flagged. I open a ticket and still no response. any help on this would be appreciated

**Comment by stevenrobson** [comment]
I am approaching 2 weeks as well. No response from support and no reason given as to why my organization was flagged in the first place. I just updated my ticket to show that I am a registered LLC and linked to my apps on Apple and Google stores.

**Reply by khemraj-h** [reply]
I just dont understand why its taking so long to get a response, I migrated a lot of my repos overs and now its giving me all sorts of issues and won't let me migrate them back

**Reply by stevenrobson** [reply]
I'm tempted to upgrade my GitHub plan for a month just to get Standard (maybe?) Support. Since I have no clue what is going on, I should probably start moving important repos over to GitLab. Just in case.

I'll report back if it helps by upgrading my GitHub plan for a month. I'll probably upgrade the first of May if my ticket continues to be disregarded. The time I've wasted just trying to resolve this issue is well over $4.00/mo.

**Reply by taine-zuse** [reply]
I am in the exact same situation. All my repos have been transferred over and me or my team cant get anything done. It blows my mind that support can sit quiet for over a week

**Reply by khemraj-h** [reply]
yeah it been over 2 weeks for me, I really dont understand how this is okay, literally no feedback

**Reply by stevenrobson** [reply]
> I am in the exact same situation. All my repos have been transferred over and me or my team cant get anything done. It blows my mind that support can sit quiet for over a week

Is your team on a paid plan?

**Reply by taine-zuse** [reply]
> > I am in the exact same situation. All my repos have been transferred over and me or my team cant get anything done. It blows my mind that support can sit quiet for over a week
> 
> Is your team on a paid plan?

Yes it is, iv paid for 3 seats but it wont even allow me to add team members so they sitting empty, basically paying for the inconvenience at this point

**Reply by stevenrobson** [reply]
> yeah it been over 2 weeks for me, I really dont understand how this is okay, literally no feedback

Any updates? I'm going on day 18. I wonder if their ticket system has a bugged.

**Reply by khemraj-h** [reply]
26 days and still not feedback

**Reply by taine-zuse** [reply]
Im on 14 days. I upgraded to enterprise in hopes of getting better support. This enabled me to create a new organisation under the enterprise however the the original organisation I have paid for is still broken. No support response from both a PAID for organisation with empty seats AS WELL as enterprise support. I am baffled

**Reply by khemraj-h** [reply]
what really gets me is I have no idea what I did or how to fix it

**Reply by stevenrobson** [reply]
Flagging an organization, without any details as to why, is a security issue itself. It is a poor security model when a service does not inform their customers when they think they have potentially detected malicious activity, security issues, or any other variables triggering their flag bot.

**Reply by khemraj-h** [reply]
did anyone get a response yet?? I am still waiting. BEEN OVER A MONTH. I am really considering migrating all my projects off of GitHub

**Reply by stevenrobson** [reply]
39 days of absolute silence. I've kinda given up. I think their layoffs earlier this year fractured their customer support services. I'm not keeping business projects that I publish on here anymore. GitLab looks promising, but I'm now using AWS CodeCommit for my production code since I already have an account and backend services running there.

They should give us badges for having unanswered support tickets older than 30+ days.

**Reply by stevenrobson** [reply]
fyi - Support finally fixed the issue. They stated that a script mistakenly flagged the account, and that the long delays for support are due to a high volume of tickets that they have. I think they should at least higher two more people in support.  :-/

**Reply by khemraj-h** [reply]
yeah they addressed my concern too, admit it was flagged incorrectly. really wish they up the support staff

**Reply by LuvIzABtfPain** [reply]
Yeah My account has been flagged for 1 week and still gotten no response yet

**Reply by microhobby** [reply]
oh boy, my organization was flagged yesterday. I think I'll start already migrating to another service then 😢. Did you guys received some e-mail about it? I saw that an e-mail should be sent from Github with more info, but I did not received nothing.

**Reply by laurensius** [reply]
i faced the same thing. without any notification from github. really confusing. and some of my repositories appear to be missing from there. how to solve this problem?

**Reply by pablo67340** [reply]
Yep, I got an email saying suspicious activity with my account, so I was forced to change my password and whatnot. When I logged in, my org was flagged. It's been a month since I messaged their appeal form, no luck. So sad to see GitHub really not give a care about an issue like this, especially when it can grossly impact a company's day to day operations. Onto another service.
